### PR TITLE
plhashmap tests

### PR DESCRIPTION
**Pull request by vektor8891**
changelist: #21 add unit tests

### CODE REVIEW

File: src/data/hashMap/PLHashMap.spec.ts [hunk 1]
```diff
@@ -20,7 +20,10 @@ describe('stdlib/data/PLHashMap', () => {
     it('should return with the JS representation', () => {
       const js = plHashMap(plString('a'), plNumber(1), plString('b'), plNumber(2)).toJS()
       expect(Array.from(js.keys())).toEqual(['a', 'b'])
-      expect(Array.from(js.values())).toEqual([1, 2])
+      expect(Array.from(js.values())).toEqual([
```
Comment (maxinteger) [inline src/data/hashMap/PLHashMap.spec.ts:23]: this change is still not valid

**Review by maxinteger — CHANGES_REQUESTED** [review]
I see 9 commits instead of 1
please synck your repo properly before you create new commit and PR
see more info here: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork

I recommend to delete this PR
sync the repos and then create a new one

File: src/data/hashMap/PLHashMap.spec.ts (hunk 1 above)
Comment (vektor8891) [reply src/data/hashMap/PLHashMap.spec.ts:23]: I tried to run the code in the link but it didn't work. This is my history:

kép

I will try to revert the changes base on [this](https://stackoverflow.com/questions/1463340/how-to-revert-multiple-git-commits).

**Comment by vektor8891** [comment]
@maxinteger I created PR #24 with a clean history.

**Pull request closed without merging**